common/define-grammar.js

```diff
@@ -834,6 +834,7 @@ module.exports = function defineGrammar(dialect) {
         'boolean',
         'string',
         'symbol',
+        'unique symbol',
```

Comment:
Can there be multiple spaces in between unique and symbol? Possibly this is better:
```suggestion
        alias(seq('unique', 'symbol'), 'unique symbol')
```

Reply:
Yes, good point, I'll use your version instead! I did feel like it was a bit fishy to hard code a space, but couldn’t point my finger on why.